nisystemlink/clients/testmonitor/utilities/_dataframe_utilities.py

```diff
@@ -1,21 +1,26 @@
 from typing import Any, Callable, Dict, List, Optional
 
 import pandas as pd
-from nisystemlink.clients.testmonitor.models import Result, Step, StepProjection
+from nisystemlink.clients.testmonitor.models import (
+    Measurement,
+    Result,
+    Step,
+    StepProjection,
+)
 from nisystemlink.clients.testmonitor.utilities.constants import DataFrameHeaders
 
 
-def is_step_data_with_name_and_measurement(data: Dict[str, Any]) -> bool:
-    """Checks if a measurement is required by ensuring it has both 'name' and 'measurement' fields.
+def is_step_data_with_name_and_measurement(measurement: Measurement) -> bool:
```

Comment:
Lets simplify the name now that this operates on a Measurement
```suggestion
def has_name_and_measurement(measurement: Measurement) -> bool:
```